frontend/public/components/operator-lifecycle-manager/clusterserviceversion.tsx

```diff
@@ -167,15 +166,15 @@ export const ClusterServiceVersionDetails: React.SFC<ClusterServiceVersionDetail
               <dt>Links</dt>
               { spec.links && spec.links.length > 0
                 ? spec.links.map((link, i) => <dd key={i} style={{display: 'flex', flexDirection: 'column'}}>
-                  {link.name} <OverflowLink value={link.url} href={link.url} />
+                  {link.name} <a className="co-break-word" href={link.url}>{link.url || '-'}</a>
```

Comment:
If we make this change, do we we want `co-break-all`?

Not new, but most other external links open in a new window. I'd think we'd want that here, too.

Reply:
> If we make this change, do we we want co-break-all?

Lemme investigate.  It wasn't needed here, but was in #1032.

> Not new, but most other external links open in a new window. I'd think we'd want that here, too.

Good catch.  Will update.

Reply:
Updated to decorate links as external and open them in a new window.  `.co-break-word` is correct here.  PTAnotherLook, @spadgett.